sources/packages/web/src/views/student/StudentAssessmentAward.vue

```diff
@@ -91,6 +95,7 @@ export default defineComponent({
       params: {
         applicationId: props.applicationId,
         assessmentId: props.assessmentId,
+        currentAssessmentId: props.currentAssessmentId,
```

Comment:
Following the below comment would prevent from passing the parameter in the route. Please let me know if I am missing something.
https://github.com/bcgov/SIMS/pull/3332/files#r1614086520

Reply:
Updated